integration_tests/testsuite/test_logging_custom.py

```diff
@@ -48,22 +47,28 @@ def runTest(self):
                          'token is {1}, '
                          'level is {2}'.format(log_name, token, level))
 
-            self.assertTrue(self._read_log(client, log_name, token, level),
-                            'Log entry not found for posted token!')
+            project_id = test_util.project_id()
+
+            FILTER = 'resource.type = "global" AND logName = ' \
```

Comment:
unfortunately I don't think this `resource.type` filter will work. since the default is for these test applications to be deployed to GAE, their log entries get written with resource type `gae.app`, not `global`. but in general, we don't want to infer the resource of these log entries in the code (because we want to be agnostic about the environment where the tests are running). 

the only "correct" way I can think of adding this filter would be to somehow figure out what environment the test application is running in, and then add the corresponding resource.type into the filter. I'm not sure that's even possible to figure out though.